README.md

```diff
@@ -48,6 +50,123 @@ You can now chat between the instances!
 
 Coming soon...
 
+## Experimental native implementation
+
+The native Nim libp2p implementation has finally arrived. Currently, it's support is experimental and shouldn't be relied on for production use. It is however under active development and we hope to achieve a reasonable level of stability in the upcoming months, as we will be integrating it across our own set of products, such as the Nim Beacon Chain.
+
+### What to expect?
+
+The current implementation implements a bare minimum set of components in order to provide a functional and interoperable libp2p implementation. These are:
+
+- A TCP transport
+- multistream-select
+- Secio
+- Mplex
+- Identify
+- FloodSub (with gossipsub being added in the near future)
+
+This stack reflects the minimal requirements for the upcoming Eth2 implementation.
+
+### How to try it out?
+
+To run it, add nim-libp2p to your project's nimble file and spawn a node as follows:
+
+```nim
```

Comment:
better move this to a file that gets regularly compiled, and link from here

Reply:
This is just a quick example, it's usually better to have it as visible as possible. A more complete example is the chat app, which is linked and built.